recipes-asteroid/asteroid-calendar/asteroid-calendar_git.bb

```diff
@@ -10,4 +10,5 @@ PV = "+git${SRCREV}"
 S = "${WORKDIR}/git"
 inherit qmake5
 
+RDEPENDS_${PN} += "qtquickcontrols-qmlplugins"
```

Comment:
This needs to be put inside a different commit. Internationalization is not the right place for that modif